.github/CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
 # The @googleapis/yoshi-python is the default owner for changes in this repo
-*               @arithmetic1728 @sai-sunder-s @TimurSadykov @googleapis/yoshi-python
+*               @arithmetic1728 @sai-sunder-s @googleapis/googleapis-auth @googleapis/yoshi-python
```

Comment:
we can remove all the individual owners now, right? no need to remove sometime later.

Reply:
We have not used this group before. So I thought it is good to test once before removing all of us.

Reply:
ok

Reply:
Could you file an issue to follow up with this? Afterwards if the group seems to be working, please remove the individual owners as it can easily be outdated, but teams are easier to maintain.